src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -68,6 +68,32 @@
  * <pre>
  * &lt;module name=&quot;EqualsAvoidNull&quot;/&gt;
  * </pre>
+ * <p>
+ * Example when ignoreEqualsIgnoreCase is set to false:
```

Comment:
1. `Example when ignoreEqualsIgnoreCase is set to false:`
It is default example, please make it just `Example:`

2. Please make it single line, e.g. <p>text</p>